src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
+@import "./header.scss";
+@import "./values.scss";
+@import "./card.scss";
+@import "./star.scss";
+
+html {
+  font-family: "Roboto", "Times New Roman", sans-serif;
+}
+
 body {
   margin: 0;
 }
```

Comment:
Consider creating a new file for fixing browser styling and importing it here to keep the main file clean